### PR TITLE
[PBIOS-278] NEW Highlight kit

### PR DESCRIPTION
**Pull request by RachelRadford21**
**What does this PR do?** A clear and concise description with your runway ticket url.
[PBIOS-278] NEW Highlight kit

**Screenshots:** Screenshots to visualize your addition/change
Simulator Screenshot - iPhone 15 Pro Max - 2024-02-16 at 13 42 07

**How to test?** Steps to confirm the desired behavior:
4. See addition/change

### Checklist
- [x] **LABELS** - Add a label: `breaking`, `bug`, `improvement`, `documentation`, or `enhancement`. See [Labels](https://github.com/powerhome/playbook-apple/labels) for descriptions.
- [x] **RELEASES** - Add the appropriate label: `Ready for Testing` / `Ready for Release`
- [x] **TESTING** - Have you tested your story?

### CODE REVIEW

File: Sources/Playbook/Components/Highlight/HighlightCatalog.swift [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    var body: some View {
+      ScrollView {
+        VStack(spacing: Spacing.medium) {
+          PBDoc(title: "Default") {
+            defaultView
+          }
+        }
+        .padding(Spacing.medium)
+      }
+      .background(Color.background(Color.BackgroundColor.light))
```
Comment (isismsilva) [inline Sources/Playbook/Components/Highlight/HighlightCatalog.swift:22]:
```suggestion
      .background(Color.background(.light))
```

File: Sources/Playbook/Components/Highlight/PBHighlight.swift [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+//
+
+import SwiftUI
+
+public struct PBHighlight: View {
+  let text: String
+  let highlightedText: [String]
+  public init(
+    text: String = "",
+    highlightedText: [String] = [""]
```
Comment (isismsilva) [inline Sources/Playbook/Components/Highlight/PBHighlight.swift:17]:
```suggestion
    highlightedText: [String] = []
```

File: Sources/Playbook/Components/Highlight/docs/_highlight_default_swift.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+```swift
+VStack(alignment: .leading, spacing: Spacing.small) {
+      PBHighlight(
```
Comment (isismsilva) [inline Sources/Playbook/Components/Highlight/docs/_highlight_default_swift.md:3]: I think the indentation looks off here

Comment (RachelRadford21) [reply Sources/Playbook/Components/Highlight/docs/_highlight_default_swift.md]: I changed the indentation idk if it's better or not. I did control + i in Xcode and that was the indentation it used(in the first example that you commented on) so just let me know if its still off!

Comment (isismsilva) [reply Sources/Playbook/Components/Highlight/docs/_highlight_default_swift.md]: Yes. It's still off. I think ctrl+i doesn't work with .md files

Comment (isismsilva) [reply Sources/Playbook/Components/Highlight/docs/_highlight_default_swift.md]: Our code convention is to use 2 spaces.

**Review by isismsilva — APPROVED** [review]

**Pull request merged**